src/partition/sph-partition-matches.cc

```diff
@@ -293,9 +302,13 @@ void Worker::defineOptions(po::options_description& opts) {
                        "chunk info is available for the identifier.");
     part.add_options()("part.chunk-mismatch-abort", po::bool_switch()->default_value(false),
                        "Abort processing if the chunk numbers found in the index maps for "
-                       "the matched identifiers aren't the same. Otherwise just complain "
+                       "the matched identifiers are not the same. Otherwise just complain "
                        "and assume that the matched objects are still within the specified "
                        "overlap radius.");
+    part.add_options()("part.position-mismatch-ignore", po::bool_switch()->default_value(false),
+                       "Do not abort processing if positions of the matched objects are not "
+                       " within the specified overlap overlap radius. Just issue the warning "
```

Comment:
typo "overlap overlap"

Reply:
Will fix it now.

Reply:
And, by the way, I'm going to add the following to the documentation of the parameter:
```
WARNING: do not use this option in production as it is meant to be used for debugging the incorrect input files.
```
A similar warning will be added to the parameter:
```
--part.position-mismatch-ignore
```
The original version of the partitioner was way too strict in this area. I added these parameters to allow debugging the inputs if it has problems. W/o these options the partitioner would choke on the very first incorrect line and won't let me see all incorrect lines in the input file(s).